### PR TITLE
Implement Book Service

### PR DESCRIPTION
**Pull request by ABaidin**

### CODE REVIEW

File: books/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
Comment (olena4reunited) [inline books/admin.py]: Add registration for model in admin

File: books/models.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+class Book(models.Model):
+    COVER_CHOICES = [
+        ("HARD", "Hardcover"),
+        ("SOFT", "Softcover"),
+    ]
+
+    title = models.CharField(max_length=255)
+    author = models.CharField(max_length=255)
+    cover = models.CharField(max_length=255, choices=COVER_CHOICES)
```
Comment (olena4reunited) [inline books/models.py]: No need for 255 max_length, 4 is OK

File: books/serializers.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from rest_framework import serializers
+from books.models import Book
+
+
+class BookSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Book
+        fields = ("title", "author", "cover", "inventory", "daily_fee")
```
Comment (olena4reunited) [inline books/serializers.py]: Add id

File: library_service/urls.py [hunk 4]
```diff
@@ -15,8 +15,12 @@
...
-from django.urls import path
+from django.urls import path, include
+from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
+    path("admin/", admin.site.urls),
+    path("api/", include("books.urls", namespace="books")),
+    path("api/token/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
+    path("api/token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
```
Comment (olena4reunited) [inline library_service/urls.py]: Remove it, tokens must be placed into user app. I know that implementation of some of your functionality based on token auth, but it shouldn't be placed here

**Review by olena4reunited — CHANGES_REQUESTED** [review]

File: books/admin.py [hunk 5]
```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
 
-# Register your models here.
+from books.models import Book
+
```
Comment (olena4reunited) [inline books/admin.py]: Should be 2 lines between imports and register

**Review by olena4reunited — CHANGES_REQUESTED** [review]

**Review by olena4reunited — APPROVED** [review]

**Review by pashawarganov — APPROVED** [review]

**Review by yashkunn — APPROVED** [review]

**Pull request merged**